lib/galaxy/container/docker.py

```diff
@@ -0,0 +1,174 @@
+"""
+Interface to Docker
+"""
+from __future__ import absolute_import
+
+import logging
+try:
+    from shlex import quote
+except ImportError:
+    from pipes import quote
```

Comment:
Can you use this ``from six.moves import shlex_quote`` instead and in the other place these lines appear.